tests/test_client.py

```diff
@@ -208,3 +212,33 @@ def test_client_error_including_field_response(client, response):
         client.payments.create(**params)
     assert excinfo.match('The amount is higher than the maximum')
     assert excinfo.value.field == 'amount'
+
+
+@pytest.mark.skipif(sys.version_info.major != 2, reason='output differs for python 2')
+def test_client_unicode_error_py2(client, response):
+    """An error response containing Unicode characters should also be processed correctly."""
+    response.post('https://api.mollie.com/v2/orders', 'order_error', status=422)
+    with pytest.raises(UnprocessableEntityError) as err:
+        # actual POST data for creating an order can be found in test_orders.py
+        client.orders.create({})
+
+    # handling the error should work even when utf-8 characters are in the response.
+    exception = err.value
+    expected = 'Order line 1 is invalid. VAT amount is off. ' \
+        'Expected VAT amount to be 3.47 (21.00% over 20.00), got 3.10'
+    assert str(exception) == expected
+
+
+@pytest.mark.skipif(sys.version_info.major == 2, reason='output differs for python 2')
+def test_client_unicode_error_py3(client, response):
+    """An error response containing Unicode characters should also be processed correctly."""
+    response.post('https://api.mollie.com/v2/orders', 'order_error', status=422)
+    with pytest.raises(UnprocessableEntityError) as err:
+        # actual POST data for creating an order can be found in test_orders.py
+        client.orders.create({})
+
+    # handling the error should work even when utf-8 characters are in the response.
```

Comment:
Took me a little bit to spot the euro sign being gone. Maybe add that to the comment, e.g.
```
# handling the error should work even when utf-8 characters (€) are in the response.
```